### DISCUSSION TITLE
Newman - ALLURE Report -Can we set dynamic data in Metadata in POSTMAN.

### DISCUSSION THREAD

**Discussion by matifniaz** [allure-framework/allure2#2369] [General Discussion]
**Can we set dynamic data in Metadata in POSTMAN?**

// @allure.label.story=storyName
// @allure.id=228
// @allure.label.{{labelName}}={{labelValue}}
// @allure.label.story=storyName
// @allure.label.suite=suiteName
// @allure.label.owner=ownerName
// @allure.label.tag=tagName

**Comment by vovsemenv** [comment]
For now that not possible. Add feature request to allure-js with your use-case and maybe we prioritize that

**Reply by matifniaz** [reply]
Thanks @vovsemenv Can you add this feature on priority
